runner/src/indexer/indexer.ts

```diff
@@ -132,7 +136,17 @@ export default class Indexer {
     return blockHeight.toString().padStart(12, '0');
   }
 
-  async fetchStreamerMessage (blockHeight: number): Promise<{ block: any, shards: any[] }> {
+  async fetchStreamerMessage (blockHeight: number, isHistorical: boolean): Promise<{ block: any, shards: any[] }> {
+    if (!isHistorical) {
+      const cachedMessage = await this.deps.redisClient.getStreamerMessageFromCache(`near-lake-data-${this.network}`, blockHeight);
+      if (cachedMessage) { // Cache hit on streamer message
+        METRICS.CACHE_HIT_STREAMER_MESSAGE.labels(isHistorical ? 'historical' : 'realtime').inc(); // increment the cache hit counter
```

Comment:
```suggestion
        METRICS.CACHE_HIT_STREAMER_MESSAGE.labels(isHistorical ? 'historical' : 'real-time').inc(); // increment the cache hit counter
```
We use this for other metrics, let's stay consistent with that.